setup.cfg

```diff
@@ -9,5 +9,6 @@ line_length = 88
 multi_line_output = 3
 use_parentheses = True
 lines_after_imports = 2
-known_third_party = PIL,bs4,classy_vision,cv2,detectron2,fairscale,faiss,fvcore,hydra,mock,nbconvert,nbformat,numpy,omegaconf,parameterized,pkg_resources,pycocotools,recommonmark,run_distributed_engines,scipy,setuptools,sklearn,sphinx_rtd_theme,tabulate,torch,torchvision,tqdm,utils
```

Comment:
is it possible to revert this?

Reply:
Yes, but since that was generated by `seed-isort-config` it seemed better to remove at, as directed by the [`isort` documentation](https://pycqa.github.io/isort/docs/configuration/pre-commit/#seed-isort-config).